lib/cartopy/tests/mpl/test_crs.py

```diff
@@ -44,17 +44,6 @@ def test_lambert_south():
     ax.gridlines()
 
 
-@pytest.mark.natural_earth
-@ImageTesting(['mercator_squashed'])
-def test_mercator_squashed():
```

Comment:
I am pretty sure this was removed because of the `ax.coastlines()` call and failing because of PROJ's new checks for transforming between incompatible globes.

Reply:
Ok